vresutils/load.py

```diff
@@ -137,35 +137,47 @@ def timeseries_opsd(years=slice("2011", "2015"), fn=None):
         load = load.loc[years]
 
     # manual alterations:
-    # GB input given in 3 regions, there is no "global one":
+    load = manual_alternations_opsd(load)
+
+    return load
+
+def copy_timeslice(load, cntry, start, stop, delta):
+    start = pd.Timestamp(start)
+    stop = pd.Timestamp(stop)
+    if start in load.index and stop in load.index:
+        load.loc[start:stop, cntry] = load.loc[start+delta:stop+delta, cntry].values
+    return load
+    
+
+def manual_alternations_opsd(load):
```

Comment:
```suggestion
def manual_alterations_opsd(load):
```

https://www.merriam-webster.com/dictionary/alteration

Reply:
Thanks...